homeassistant/components/sensor/recorder.py

```diff
@@ -678,7 +678,9 @@ def validate_statistics(
 
     for state in sensor_states:
         entity_id = state.entity_id
-        state_class = state.attributes.get(ATTR_STATE_CLASS)
+        state_class: SensorStateClass | None = None
+        with contextlib.suppress(ValueError):
+            state_class = SensorStateClass(state.attributes.get(ATTR_STATE_CLASS))  # type: ignore[arg-type]
```

Comment:
Does this work?
```suggestion
            state_class = SensorStateClass(str(state.attributes.get(ATTR_STATE_CLASS)))
```

Reply:
We should not copy `None` to string just to be able to remove the type ignore.

Reply:
Oh! then I misunderstood what @frenck has been recommending.
See https://github.com/home-assistant/core/blob/ad87c4557d25a3fb68543afe5450cb2e10d52f34/homeassistant/components/sensor/__init__.py#L467-L471

Reply:
If #87082 is approved, this can be reduced